lib/modules/apostrophe-ui/public/css/components/tooltip.less

```diff
@@ -0,0 +1,35 @@
+.apos-tooltip
```

Comment:
I am. It looks off from house style which you can reference here https://github.com/punkave/stylelint-config-punkave/blob/master/index.js

But I wouldn't sweat it too much and definitely stick to spacing patterns in apos.